### PR TITLE
rename backup-metadata to backup-metadata-generator

### PR DESCRIPTION
**Pull request by tcdowney**
CI changes for https://github.com/cloudfoundry/capi-k8s-release/pull/105/

CAKE Story: [#175080281](https://www.pivotaltracker.com/story/show/175080281)

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -21,14 +21,14 @@ groups:
   - cf-api-controllers-unit-tests
   - registry-buddy-unit-tests
   - registry-buddy-integration-tests
-  - backup-metadata-unit-tests
-  - build-backup-metadata-image
+  - backup-metadata-generator-generator-unit-tests
+  - build-backup-metadata-generator-generator-image
```
Comment (matt-royal) [inline ci/pipeline.yml:25]: I'm guessing this is a typo and should just be `generator-`, not `generator-generator-`

**Review by matt-royal — CHANGES_REQUESTED** [review]
It looks like there's a typo, which I commented on (e.g. `generator-generator-` instead of just `generator-`). Other than that this looks good.

File: ci/pipeline.yml (hunk 1 above)
Comment (tcdowney) [reply ci/pipeline.yml:25]: yes, thank you for catching this 🙇‍♂️

Comment (tcdowney) [reply ci/pipeline.yml]: fixed and amended the commit
https://github.com/cloudfoundry/capi-ci/pull/16/commits/5ea7abc3e0c99eda0b702ee3db1e96de39a80c3f

**Pull request merged**